.github/workflows/run-example-deployment.yml

```diff
@@ -0,0 +1,23 @@
+name: 'GitHub Action deploy example'
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  run-example-deployment:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+#        with:
+#          path: "github-action-deploy-on-vercel" # Act doesn't work if not specifying the path. See https://github.com/nektos/act/issues/228
```

Comment:
@Demmonius When specified, GHA fails. See https://github.com/UnlyEd/github-action-deploy-on-vercel/actions/runs/617433865

When commented, it passes.

What should we do about that? Is this supposed to be run through Act only? (locally)

Reply:
I'll keep it disabled for now until I get feedback from you